src/components/AssetsStatusAlert/index.jsx

```diff
@@ -0,0 +1,228 @@
+  }
+
+  updateStatusAlertFields(assetsStatus, deletedAsset) {
+    const assetName = deletedAsset.display_name;
+    let alertDialog;
+    let alertType;
+
+    const genericUpdateError = (
+      <WrappedMessage
+        message={messages.assetsStatusAlertGenericUpdateError}
+      />
+    );
+
+    switch (assetsStatus.type) {
+      case assetActions.delete.DELETE_ASSET_FAILURE:
+        alertDialog = (
+          <WrappedMessage
+            message={messages.assetsStatusAlertCantDelete}
+            values={{ assetName }}
+          />
+        );
+        alertType = 'danger';
+        break;
+      case assetActions.delete.DELETE_ASSET_SUCCESS:
+        alertDialog = (
+          <WrappedMessage
+            message={messages.assetsStatusAlertDeleteSuccess}
+            values={{ assetName }}
+          />
+        );
+        alertType = 'success';
+        break;
+      case assetActions.upload.UPLOAD_ASSET_SUCCESS:
+        this.updateUploadSuccessCount();
+        alertDialog = (
+          <WrappedMessage
+            message={messages.assetsStatusAlertUploadSuccess}
+            values={{ uploaded_count: (<FormattedNumber value={this.state.uploadSuccessCount} />) }}
+          />
+        );
+        alertType = 'success';
+        break;
+      case assetActions.upload.UPLOADING_ASSETS:
+        this.closeStatusAlert();
+        alertDialog = (
+          <WrappedMessage
+            message={messages.assetsStatusAlertUploadInProgress}
+            values={{ uploading_count: (<FormattedNumber value={assetsStatus.count} />) }}
+          />
+        );
+        alertType = 'info';
+        break;
+      case assetActions.upload.UPLOAD_EXCEED_MAX_COUNT_ERROR:
+        alertDialog = (
+          <WrappedMessage
+            message={messages.assetsStatusAlertTooManyFiles}
+            values={{ max_count: (<FormattedNumber value={assetsStatus.maxFileCount} />) }}
+          />
+        );
+        alertType = 'danger';
+        break;
+      case assetActions.upload.UPLOAD_EXCEED_MAX_SIZE_ERROR:
+        alertDialog = (
+          <WrappedMessage
+            message={messages.assetsStatusAlertTooMuchData}
+            values={{ max_size: (<FormattedNumber value={assetsStatus.maxFileSizeMB} />) }}
+          />
+        );
+        alertType = 'danger';
+        break;
+      case assetActions.upload.UPLOAD_ASSET_FAILURE:
+        alertDialog = (
+          <WrappedMessage
+            message={messages.assetsStatusAlertGenericError}
+            values={{ assetName: assetsStatus.asset.name }}
+          />
+        );
+        alertType = 'danger';
+        break;
+      case assetActions.lock.TOGGLING_LOCK_ASSET_FAILURE:
+        alertDialog = (
+          <WrappedMessage
+            message={messages.assetsStatusAlertFailedLock}
+            values={{ assetName: assetsStatus.asset.name }}
+          />
+        );
+        alertType = 'danger';
+        break;
+      case assetActions.clear.CLEAR_FILTERS_FAILURE:
```

Comment:
Can these next four `case`s be combined?

```javascript
case assetActions.clear.CLEAR_FILTERS_FAILURE:
case assetActions.filter.FILTER_UPDATE_FAILURE:
```

Reply:
Yes. Thank you. I tried to figure out how to combine cases but wasn't able to find the correct way to do it online.